src/components/course/CourseHeader.jsx

```diff
@@ -98,12 +93,44 @@ export default function CourseHeader() {
                     <img
                       src={partner.logoImageUrl}
                       alt={`${partner.name} logo`}
-                      style={{ maxWidth: 160 }}
+                      style={{ maxWidth: 160, maxHeight: 44 }}
                     />
                   </a>
                 ))}
               </div>
             )}
+            <div className={classNames({ 'mb-4': !course.shortDescription })}>
+              <h2>{course.title}</h2>
+            </div>
+            {course.shortDescription && (
+              <div
+                className="lead font-weight-normal mb-4"
+                // eslint-disable-next-line react/no-danger
+                dangerouslySetInnerHTML={{ __html: course.shortDescription }}
+              />
+            )}
+            {course.skillNames.length > 0 && (
+              <div className="mb-4">
+                <h6> Skills you&apos;ll gain</h6>
+                <div>
+                  {course.skillNames.map((skill, index) => (
+                    <Badge
+                      key={skill.id}
+                      className="course-badge"
+                      variant="light"
+                      style={{ display: index < maxVisibleSkills || showMore ? 'inline-block' : 'none' }}
+                    >
+                      {skill}
+                    </Badge>
+                  ))}
+                  {course.skillNames.length > maxVisibleSkills ? (
```

Comment:
style suggestion: instead of conditionally rendering null you can also do a slightly more succinct pattern of 

```
(course.skillNames.length > maxVisibleSkills) && (whateverToRenderConditionally)
```

Reply:
Fixed.